pkg/executor/push_test.go

```diff
@@ -57,21 +57,24 @@ func TestDockerConfLocation(t *testing.T) {
 	if unset != unsetExpected {
 		t.Errorf("Unexpected default Docker configuration file location: expected:'%s' got:'%s'", unsetExpected, unset)
 	}
-
-	if err := os.Setenv(dcfg, "/kaniko/.docker"); err != nil {
+	tmpDir, _ := ioutil.TempDir("", "*")
```

Comment:
please add a defer to clean up this temporary directory.

Reply:
done